core/roboconf-target-iaas-vmware/src/main/java/net/roboconf/target/vmware/internal/VmwareIaasHandler.java

```diff
@@ -90,10 +91,19 @@ public String createMachine(
 			String messagingIp,
 			String messagingUsername,
 			String messagingPassword,
-			String rootInstanceName,
+			String scopedInstancePath,
 			String applicationName )
 	throws TargetException {
 
+		this.logger.fine( "Creating a new VM @ VMware." );
+
+		// For IaaS, we only expect root instance names to be passed
+		if( InstanceHelpers.countInstances( scopedInstancePath ) > 1 )
+			throw new TargetException( "Only root instances can be passed in arguments." );
```

Comment:
Not sure, but VMware might be deployed inside a VM ?? If so, should be a warning rather than an exception.